pkgs/development/python-modules/pytensor/default.nix

```diff
@@ -63,8 +65,9 @@ buildPythonPackage rec {
     jax
     jaxlib
     numba
-    pytestCheckHook
+    pytest-benchmark
```

Comment:
Do we really want to run benchmarks? They require lots of resources for not much benefits.